src/services/index.js

```diff
@@ -0,0 +1,4 @@
+const {
+  createRecordService,
+  readRecordService,
+  } = require('./recordService');
```

Comment:
```suggestion
const recordService = require('./recordService');
  
  module.exports = {
    recordService
  }
```

모듈을 require 받아서 package로 보내기 위해 사용하는 구문이기 때문에, 모듈(파일)별로 require하고 exports 하시면 됩니다.

Reply:
반영했습니다.